elastic/integrations
# [MongoDb]Updated host.hostname field use FQDN of the host

- Bug


## What does this PR do?

This PR adds the FQDN of the host and refactored the default pipeline.
## Checklist


## How to test this PR locally

- Clone integrations repo.
- Install elastic-package locally.
- Configure the mongodb log file path in Integration UI.
- log data should flow in to the kibana.


## Related issues

- Closes #761
